app/react/Library/LibraryTable.js

```diff
@@ -0,0 +1,22 @@
+import RouteHandler from 'app/App/RouteHandler';
+import React from 'react';
+import { TableViewer } from 'app/Layout/TableViewer';
+import Library from 'app/Library/Library';
+import requestState from 'app/Library/helpers/requestState';
+
+export class LibraryTable extends RouteHandler {
+  static async requestState(requestParams, globalResources) {
+    return requestState(requestParams, globalResources, true);
+  }
+
+  render() {
+    return (
+      <Library
+        viewer={TableViewer}
+        location={this.props.location}
+        fixedSidePanels
+        showFiltersButton
+      />
```

Comment:
This is the place to say: "show the column selector", "pass the side panel a table-view-mode class".

Not the assumption of: "if I have viewer, put the column selector", "if I passed fixedSidePanels, put a table-view-mode".   

It could be as explicit as: "sidepanelClassName="table-view-mode", and you pass that to the side panel on libraryLayout, for example.

Reply:
Yes, is part of the same thing. :+1: